APIScrape/README.md

```diff
@@ -0,0 +1,85 @@
+# API Scraper
+
+A command-line utility for creating the database for the Top Cat app.
+
+* Makes API requests to get:
+    * A list of cat breeds
+    * Image URLs for cats from each breed
+* **To run the script your mySQL server must already have a database named `cat-test`!!**
```

Comment:
I don't think this matches the code below?

Reply:
It does, because of a dependency on DB.php.

When DB.php makes the PDO connection it throws up if it can't find a database named `cat-test`, because of line 5 specifying the dbname. 

DB.php could be changed but I'm not sure how this affects the app & team determined we would add this line to the readme instead.

Reply:
Hm. We don't need a database to connect to MySQL. How about changing the DB class so we can do both/either?

Reply:
Agree but want to know the team agrees on the way forward.

Reply:
Team has decided to create another function in DB.php to connect to mySQL without specifying dbname, to be used specifically in the scrape script, so it can do its database setup stuff.

Will do this tomorrow.